### PR TITLE
Add instructions to keep private key secure

### PR DESCRIPTION
**Pull request by yoyothesheep**
Add explicit instructions for .gitignore, so users don't commit their private key by accident.

### CODE REVIEW

File: NFT_Collection/en/Section_1/Lesson_4_Create_A_Contract_That_Mints_NFTs.md [hunk 1]
```diff
@@ -307,6 +307,11 @@ You can grab your API URL from the Alchemy dashboard and paste that in. Then, yo
 
 **Note: DON'T COMMIT THIS FILE TO GITHUB. IT HAS YOUR PRIVATE KEY. YOU WILL GET HACKED + ROBBED. THIS PRIVATE KEY IS THE SAME AS YOUR MAINNET PRIVATE KEY.** We'll talk about `.env` variables later and how to keep this stuff secret.
 
+In the meantime - open your `.gitignore` file and add a line for hardhat.config.js. You can remove this later when you set up `.env`.
```
Comment (adilanchian) [inline NFT_Collection/en/Section_1/Lesson_4_Create_A_Contract_That_Mints_NFTs.md:310]: Can you add backticks around hardhat.config.js please?

**Review by adilanchian — CHANGES_REQUESTED** [review]
this is probably a smart move !

One thing I would say as well is when we talk about this `.env` file, can you add a call out to this?

File: NFT_Collection/en/Section_1/Lesson_4_Create_A_Contract_That_Mints_NFTs.md (hunk 1 above)
Comment (yoyothesheep) [reply NFT_Collection/en/Section_1/Lesson_4_Create_A_Contract_That_Mints_NFTs.md]: Done

**Comment by yoyothesheep** [comment]
OK updated per your comments, pls take another look

**Comment by adilanchian** [comment]
Hey @yoyothesheep -

Looks great! Thanks so much for the changes! 

LGTM

**Comment by adilanchian** [comment]
ALSO - CONGRATS ON YOUR FIRST CONTRIBUTION

🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

**Comment by adilanchian** [comment]
@all-contributors please add @yoyothesheep  for doc

**Pull request merged**